browse/templates/user_banner.html

```diff
@@ -1,3 +1,5 @@
+{# donation banner #}
+{%- if 0 -%}
```

Comment:
Wasn't this part of an earlier PR too? Or maybe this isn't important because that is already in develop?

Reply:
I don't think so?

Reply:
Oh I see, my bad!